src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected Integer index;
+
+    public Task(String description, boolean b, int i) {
+        this.description = description;
+        this.isDone = b;
+        this.index = i;
+    }
```

Comment:
Perhaps more intuitive variable names would be isDone and index?
```suggestion
    public Task(String description, boolean isDone, int index) {
        this.description = description;
        this.isDone = isDone;
        this.index = index;
    }
```

Reply:
I agree with this comment, will be easier for readability as well as making changes to the code in the future